src/Logging.AzureAppServices/src/AzureBlobLoggerOptions.cs

```diff
@@ -30,6 +29,8 @@ public string BlobName
             }
         }
 
+        internal string CustomPrefixFileName { get; set; }
```

Comment:
Suggest moving "Prefix" to the end because it seems more natural
```suggestion
        internal string CustomFileNamePrefix { get; set; }
```

Side note: Wish we were consistent about "filename" (more common) vs. "fileName" and "Filename" (less common) vs. "FileName" in this repo.

Reply:
- Changed to CustomFileNamePrefix